File: CS/App1/ViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace App1
{
    public class ViewModel : ViewModelBase
    {
        
        protected ObservableCollection<User> _Users;
        public ObservableCollection<User> Users
        {
            get
            {
                if (this._Users == null)
                {
                    this._Users = new ObservableCollection<User>() {
                        new User(0, "Jack", UserRole.Administrator),
                        new User(1, "Ron", UserRole.User),
                        new User(2, "John", UserRole.User),
                        new User(3, "Antoni", UserRole.User),
                        new User(4, "Paul", UserRole.Moderator),
                    };

                }

                return this._Users;
            }
        }


        protected UserRole _SelectedRole;
        public UserRole SelectedRole
        {
            get { return this._SelectedRole; }
            set { this.SetProperty(ref this._SelectedRole, value, "SelectedRole"); }
        }

    }
}

File: VB/App2/ViewModel.vb
Imports DevExpress.Mvvm

Public Class ViewModel
    Inherits ViewModelBase

    Protected _Users As ObservableCollection(Of User)

    Public ReadOnly Property Users As ObservableCollection(Of User)
        Get

            If Me._Users Is Nothing Then
                Me._Users = New ObservableCollection(Of User)() From {
                    New User(0, "Jack", UserRole.Administrator),
                    New User(1, "Ron", UserRole.User),
                    New User(2, "John", UserRole.User),
                    New User(3, "Antoni", UserRole.User),
                    New User(4, "Paul", UserRole.Moderator)
                }
            End If

            Return Me._Users
        End Get
    End Property

    Protected _SelectedRole As UserRole

    Public Property SelectedRole As UserRole
        Get
            Return Me._SelectedRole
        End Get
        Set(ByVal value As UserRole)
            Me.SetProperty(Me._SelectedRole, value, "SelectedRole")
        End Set
    End Property
End Class
